test/functional/interface_bitcoin_cli.py

```diff
@@ -27,6 +28,41 @@
 WALLET_NOT_LOADED = 'Requested wallet does not exist or is not loaded'
 WALLET_NOT_SPECIFIED = 'Wallet file not specified'
 
+
+def cli_get_info_string_to_dict(cli_get_info_string):
+    # helper method to convert human readable -getinfo into a dictionary
```

Comment:
```suggestion
    """Helper method to convert human readable -getinfo into a dictionary"""
```

Reply:
👍 Resolved it in [`b634ff2`](https://github.com/bitcoin/bitcoin/commit/b634ff26ab8cf4bdffed1a75300ec63a9da22cf1)